portal-kernel/test/unit/com/liferay/portal/modules/ModulesStructureTest.java

```diff
@@ -30,10 +30,12 @@
 import java.io.FileReader;
 import java.io.IOException;
 import java.io.InputStream;
-
```

Comment:
SF will complain here AFAIK. Did you run `ant format-source` from `portal-impl`?